### PR TITLE
Update README

### LINKED ISSUE #123: Update README
- Update notice
- Add notes for contributions

### PR DESCRIPTION
**Pull request by NoB0**
- Update Notice
- Add contributions guidelines
- Add publication citation

Link to convention guidelines will be updated once the related PR is merged.

Fixes #123

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# :warning: Notice
+# :warning: Disclaimer
 
-> It appears that you have stumbled upon our GitHub repository as a result of our publication being [released on arXiv](https://arxiv.org/abs/2301.05544) ahead of schedule. While you are welcome to look around, please be advised that the code is not yet fully functional as we have been making a few breaking changes. We kindly ask that you check back at a later time for updates. Be assured, by the time of the WSDM conference, the code will be working as described and this message will be gone. Thank you for your understanding!
+> The code is currently undergoing some refactoring, so the simulation is not working as describe in the paper. We kindly ask for some more of your patience. Be assured, we are doing our best to finish this refactoring as quickly as possible. Thank you for your understanding!
```
Comment (kbalog) [inline README.md:3]: Nit: "as descibed"

**Review by kbalog — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (NoB0) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -74,12 +74,23 @@ The simulated dialogue is saved under `dialogue_export`.
...
-  * Jafar Afzali (2022)
-  * Krisztian Balog (2021-present)
-  * Nolwenn Bernard (2022-present)
-  * Aleksander Drzewiecki (2022)
-  * Shuo Zhang (2021)
+We welcome contributions both on the high level (feedback and ideas) as well as on the more technical level (pull requests). See our [contribution guidelines](https://github.com/iai-group/guidelines/blob/feature/63-Add-guidelines-for-contributors/github/Contribution.md) for more details.
+
+## Publication
+
+If you are using this repository, please cite the following paper:
```
[the remark this answers is not in the document]
Comment (NoB0) [reply README.md]: Done.

**Review by kbalog — APPROVED** [review]
LGTM

The line under contributors (developed and maintained by IAI group...) was good, put that back, I just also wanted the list of individual contributors (as not all are current members of IAI).

**Pull request merged**